src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -3002,10 +2999,7 @@ impl Step for CodegenCranelift {
 
         let triple = run.target.triple;
         let target_supported = if triple.contains("linux") {
-            triple.contains("x86_64")
-                || triple.contains("aarch64")
-                || triple.contains("s390x")
-                || triple.contains("riscv64gc")
+            triple.contains("x86_64") || triple.contains("aarch64") || triple.contains("s390x")
```

Comment:
Did you mess up a rebase? I made the exact opposite change yesterday.

Reply:
I completely did...